### ISSUE TITLE
1776 Windows: Lighting + Cubemaps

### ISSUE THREAD

**Issue by tedbarnett** [TimeWalkOrg/Manhattan_public#16]
TLDR: Create a controlled but randomized system for the cube maps within the windows of each building, and the lighting intensity thats coming off of them.

**Cubemaps:**

Dario's 1776 cubemaps look great, but we will want more variety so that every window is not identical. Ideally we can include:

3-4 variants on the upstairs parlor or bedroom scenes
2-3 variants appropriate for ground-floor windows (living room, office, kitchen scenes)

To note for cubemaps: "Dutch homes had one or two rooms on the first floor, a smaller upper garret for storage, and many times a clock loft overhead. The outside doors of their houses were split horizontally. "
[(New Netherland Institute)](https://www.newnetherlandinstitute.org/education/for-teachers/what-was-new-netherland/what-were-their-homes-like)

[This document](https://docs.google.com/document/d/1lgJ_S4nJ9CSLXitm3AnV8Q9IhPH7OvGnVGdHZlOaZiE/edit?usp=sharing) has some examples of window cubemap variations, including a kitchen (example 1/2) and bedroom.

**Lighting:** 

Can we add a construction script (or equivalent) to each of the 1776 window scenes that randomly sets the intensity?  Right now, all of them are on at 100%. 

Let's make x% (say 20% to start) of the windows "off" (glass reflection only) and the remainder of linearly random intensity between 1 and 100%.

The lit windows in 1776 look a bit too uniform (every single window is lit up, and they look too bright for an era lit by gas and candles.  Can we...

- Randomly disable 40% of the windows (so that they remain "dark", looking like unlit glass windows)
- Nice to have: lights "go out" over time so that by 3am, 95% are off.  Start turning on again at 5am.
- Enable first-floor windows too (they all appear to be dark)
- Lower the brightness overall by 30% (possible to randomize the levels a bit too?)
- Make some windows "flicker" as if lit by candles or inefficient gas lighting

If these are simple parameter changes somewhere, let us know.

![image](https://github.com/TimeWalkOrg/Manhattan/assets/694013/63f50b3c-1ec4-42a2-b2ac-8ebfcc08bdf1)

**Specific Windows:**

This new building has a cool new look (and matches the Castello Map), but the windows look too identical, closely spaced (near the door, eg.) and therefore not realistic.  Can they be varied 1-2 ways easily?  Or the 2nd floor 3 windows combined into one longer one somehow?

BP_1660-A-03_Marten_Cregier

**Comment by tedbarnett** [comment]
![image](https://github.com/TimeWalkOrg/Manhattan/assets/694013/998948ef-40f6-4b55-8c5d-6b8ae817f6ae)
